javascript-carousel/main.js

```diff
@@ -0,0 +1,58 @@
+var images = [];
+images[0] = './images/001.png';
+images[1] = './images/004.png';
+images[2] = './images/007.png';
+images[3] = './images/025.png';
+images[4] = './images/039.png';
```

Comment:
Instead of assigning the image URLs one at a time, put them all in the array literal.

```js
var images = [
  './images/001.png',
  // etc
];
```